cluster/gce/gci/configure-helper.sh

```diff
@@ -2502,6 +2527,13 @@ function start-kube-addons {
   local -r src_dir="${KUBE_HOME}/kube-manifests/kubernetes/gci-trusty"
   local -r dst_dir="/etc/kubernetes/addons"
 
+  # need kube-apiserver to be ready
+  until kubectl get nodes; do
```

Comment:
We already have two places (e.g. `update-legacy-addon-node-labels()`) that do this check. Maybe make a `wait-til-apiserver-ready()` func and put that under main?

Reply:
Added.

Reply:
Thanks! Can we run `wait-till-apiserver-ready` just once so individual step won't need to check explicitly?
```bash
    start-kube-scheduler
    wait-till-apiserver-ready
    start-kube-addons
```